cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/RunnableTaskLauncher.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.app.worker;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import io.cdap.cdap.common.conf.CConfiguration;
+import io.cdap.cdap.common.internal.worker.RunnableTask;
+import io.cdap.cdap.common.internal.worker.RunnableTaskContext;
+
+/**
+ * RunnableTaskLauncher launches a {@link RunnableTask} by loading its class and calling its run method.
+ */
+public class RunnableTaskLauncher {
```

Comment:
Why this is needed? Seems like the logic is simple enough that it can be just a private method in the handler class?

Reply:
The logic will be expanded with loading classes from other jars. See [this change](https://github.com/cdapio/cdap/commit/4cbffed4f7aefd8b584b49f2cdf033640714db76). That PR will be submitted once this PR gets in.